application/src/gui/modules/settings.py

```diff
@@ -0,0 +1,149 @@
+from PySide2.QtWidgets import QMessageBox # type: ignore
+import os
+import sys
+import toml
+import functools
+from .view_type import ViewType
+from PySide2 import QtGui  # type: ignore
+from PySide2.QtCore import Qt  # type: ignore
+from PySide2.QtWidgets import QHBoxLayout  # type: ignore
+from PySide2.QtWidgets import QPushButton  # type: ignore
+from PySide2.QtWidgets import QCheckBox  # type: ignore
+from PySide2.QtWidgets import QSpinBox  # type: ignore
+from PySide2.QtWidgets import QSlider  # type: ignore
+from PySide2.QtWidgets import QVBoxLayout  # type: ignore
+from PySide2.QtWidgets import QWidget  # type: ignore
+from PySide2.QtWidgets import QLabel  # type: ignore
+
+width = 300
+height = 220
+font = 10
+opaci_percent = 100
+opaci = opaci_percent/100
+frame = False
+top = False
```

Comment:
What about importing setting instead?

`from src.utils.config`


Generally speaking if we want to introduce GUI based config. I'd suggest to create a `class` for the `Config`.

In the `utils.config` we could have module variable with `Config` instance.

You could then simply *retrieve** and **change** configurable values and save them through `Config` methods :)